jenkins/bootstrap.py

```diff
@@ -967,6 +975,23 @@ def parse_args(arguments=None):
         '--clean',
         action='store_true',
         help='Clean the git repo before running tests.')
+    # Use "_no_-magic-env" because the default (current functionality)
+    # is to have magic environment, so the default value for this flag
+    # (false) needs to match this behavior.
+    parser.add_argument(
+        '--raw-env',
+        action='store_true',
+        help='Disable Jenkins magic environment')
+    # If specified, this is the (relative) path to jobs configuration directory
+    # the cloned repo.
+    parser.add_argument(
+        '--jobs-dir',
+        # The default behavior assumes that bootstrap.py is run from the jenkins/
```

Comment:
This seems overly verbose, do you really need a whole paragraph to explain what this does?